examples/globus_app.py

```diff
@@ -83,6 +82,9 @@
 from invenio_oauthclient.views.client import blueprint as blueprint_client
 from invenio_oauthclient.views.settings import blueprint as blueprint_settings
 
+from invenio_oauthclient._compat import monkey_patch_werkzeug  # noqa isort:skip
```

Comment:
I think you forgot the monkeypatch call.

Reply:
the trick is that I actually don't have to call it, since it goes through `__init__.py` of the package and it is invoked there. You just need to make sure it is invoked, and this is what this line does

Reply:
Bu in the rest of the examples you call it, what is the difference?

Reply:
I agree with the explicit call to the function

Reply:
done